.github/workflows/eden_gcp.yml

```diff
@@ -120,17 +124,10 @@ jobs:
         run: |
           EDEN_TEST=gcp ./eden test tests/workflow -v debug
       - name: Collect info
-        if: ${{ failure() }}
-        run: |
-          # Give EVE 5 minutes at most to enable ssh access (if tests failed early).
-          for i in $(seq 60); do ./eden eve ssh && break || sleep 5; done
-          ./eden sdn fwd eth0 22 --\
-            ssh -o StrictHostKeyChecking=no -p FWD_PORT -i ./dist/default-certs/id_rsa root@FWD_IP collect-info.sh &&\
-          ./eden sdn fwd eth0 22 --\
-            scp -o StrictHostKeyChecking=no -P FWD_PORT -i ./dist/default-certs/id_rsa root@FWD_IP:/persist/eve-info-* . &&\
-          # upload-artifact complains about colon in the file name
-          mv eve-info-* eve-info.tar.gz ||\
-          echo "failed to collect info"
+        if: failure()
+        uses: ./.github/actions/collect-info
+        with:
+          info_file: ${{ env.info_file }}
```

Comment:
Should it be aligned with defined env on top of file (`INFO_LOG_FOLDER`)?

Reply:
You are correct, I removed this env variable and hard-coded paths